app/Resources/views/receipt/create_receipt.html.twig

```diff
@@ -0,0 +1,34 @@
+{{ form_start(form) }}
+    <div class="row">
+        <div class="small-12 medium-8 columns">
+            {{ form_row(form.description) }}
+        </div>
+    </div>
+
+    <div class="row">
+        <div class="small-8 medium-3 columns">
+            {{ form_row(form.sum) }}
+        </div>
+    </div>
+
+    <div class="row">
+        <div class="small-8 medium-3 columns">
+            {{ form_row(form.user) }}
+        </div>
+    </div>
+
+    <div class="row">
+        <div class="small-12 columns">
+            <div>
+                {{ form_row(form.picturePath) }}
+            </div>
+        </div>
+        <div class="small-4 columns">
+            <div class="{{ receipt.picturePath is not null ? '' : 'hide' }}">
+                <img src="{{ asset(receipt.picturePath) }}" height="60px" id="receipt_picture">
+            </div>
+        </div>
+    </div>
+    <br>
+    <button class="button success">Registrer</button>
```

Comment:
Siden personene som registrerer utlegg gjør dette fordi de ønsker å få penger tilbake, kan "Be om refusjon" matche bedre hva brukeren ønsker å oppnå enn "Registrer"?